scss/_patterns_lists.scss

```diff
@@ -1,36 +1,49 @@
 // Default list styling
 @mixin vf-lists {
   @include vf-lists-list;
-  @include vf-lists-list-ticks;
-  @include vf-lists-no-bullets;
+  @include vf-lists-list-divided;
+  @include vf-lists-list-item-state;
   @include vf-lists-combined-list;
   @include vf-lists-inline-list;
+  @include vf-lists-inline-list-middot;
   @include vf-stepped-list;
 }
```

Comment:
Can we rename the mixins with a ```-p-``` prefix so it would follow what we have done in buttons or card? For example ```@include vf-p-list``` and then it stops the lists list repetition.